### DISCUSSION TITLE
在网关中如何使用，切换不同的文档

### DISCUSSION THREAD

**Discussion by wangbenchi66** [scalar/scalar#3512] [Q&A]
在网关中配合Ocelot多项目文档，代码如何添加，页面如何进行切换

**Comment by amritk** [comment]
Hello! Sorry but do you think you could translate your question to english?

https://translate.google.com/

**Reply by wangbenchi66** [reply]
How to use it in the microservice gateway using Ocelot, and how to add multiple document addresses for document switching
